python/ql/lib/semmle/python/Flow.qll

```diff
@@ -840,7 +842,11 @@ private AstNode assigned_value(Expr lhs) {
   /* for lhs in seq: => `result` is the `for` node, representing the `iter(next(seq))` operation. */
   result.(For).getTarget() = lhs
   or
+  // def f(a = 42):  =>  `result` is 42
```

Comment:
Thanks for adding this :+1: 

I suggest we keep naming consistent with the other comments

```suggestion
  // def f(lhs = 42):  =>  `result` is 42
```